performance/README.md

```diff
@@ -0,0 +1,67 @@
+# Internal User Document: Load Test Performance Analysis Script
+
+## Introduction
+The `load_test_performance_analysis.py` script is designed to conduct load testing on SM-SDK APIs. Currently, it tests three APIs: `get_cycles`, `get_line_data`, and `get_kpi_data_viz` to analyze their performance metrics. This document provides a quick guide on how to use the script.
+
+## Prerequisites
+Before using the script, ensure the following prerequisites are met:
+
+- Python 3.8.18 and above installed on your system.
+- Necessary Python packages installed. You can install the required packages using pip:
+  ```
+  pip install -r requirements.txt
+  ```
+
+## Usage
+
+### 1. Configuration
+If a custom configuration file is provided as a command-line argument, the script will use the parameters specified in that file. Otherwise, it will use hardcoded default values for configuration parameters. The configuration file should include the following parameters:
+- `num_peak_users`: Number of peak users to simulate load.
+- `ramp_up_rate`: Rate at which users are ramped up.
+- `total_run_time`: Total duration for which the load test should run.
+- `min_wait_time`: Minimum wait time between API requests.
+- `max_wait_time`: Maximum wait time between API requests. 
+
+If any of these parameters are missing in the configuration file, the script will use hardcoded default values for them.
+
+### 2. Command Line Arguments
+The script accepts two optional command-line arguments:
+- `--config-file`: Path to the custom configuration file (default: None). If provided, the script will use the parameters specified in this file. If not provided, the script will use hardcoded default values for configuration parameters.
+- `--metrics-xml`: Path to the XML file where performance metrics will be dumped (default: None). If provided, performance metrics will be saved to this file.
+
+
+### 3. Environment Variables
+Ensure the following environment variables are set:
+- `ENV_VAR_TENANT`: Your SMSDK tenant ID.
+- `ENV_VAR_API_KEY`: Your API key.
+- `ENV_VAR_API_SECRET`: Your API secret.
+
+#### Example:
+
+```bash
+export ENV_VAR_TENANT="<Tenant Name>"
+export ENV_VAR_API_KEY="<Tenant API Key>"
+export ENV_VAR_API_SECRET="<Tenant API Secret>"
```

Comment:
Change the prefix from `ENV` to `ENV_SDK` ie `ENV_VAR_TENANT` becomes `ENV_SDK_VAR_TENANT` we had to change the env vars in the org

Reply:
Done.